test/lib/restforce/db/runner_cache_test.rb

```diff
@@ -0,0 +1,60 @@
+require_relative "../../../test_helper"
+
+describe Restforce::DB::RunnerCache do
+
+  configure!
+  mappings!
+
+  let(:cache) { Restforce::DB::RunnerCache.new }
+
+  describe "#collection" do
+    let(:object) { mapping.database_model.create! }
+
+    before { object }
+
+    it "invokes the specified collection for the mapping" do
+      instances = cache.collection(mapping, :database_record_type)
+      expect(instances.first.record).to_equal object
+    end
+
+    describe "on repeated calls for the same mapping" do
+      let(:dummy_collection) do
+        Object.new.tap do |collection|
+
+          def collection.values
+            [1, 2, 3]
+          end
+
+          def collection.each(*_)
+            values.each { |i| yield i }
+          end
+        end
+      end
+
+      before do
+        cache.collection(mapping, :database_record_type) { next }
```

Comment:
What is this `{ next }` block? It doesn't seem like `RunnerCache#collection` takes a block?